benefits/static/css/styles.css

```diff
@@ -558,27 +556,12 @@ footer .footer-links li a.footer-link:visited {
 :root {
   --radio-button-size: calc(24rem / 16);
   --radio-input-color: var(--standout-color);
-  --radio-input-gap: calc(24rem / 16);
-}
-
-@media (min-width: 992px) {
-  :root {
-    --radio-input-gap: calc(15rem / 16);
-  }
 }
 
 .radio-label {
   cursor: pointer;
 }
 
-.radio-input-group {
-  gap: var(--radio-input-gap);
-}
```

Comment:
This is a refactor that I made by:

- Bumping up the 15px gap on desktop to 16px,
- Allows us to get to place where we are only using 8pt values for both desktop (16px) and mobile (24px)
- Which then allows us to use `gap-4` (4 is 24px) and `gap-md-3` (3 is 16px) in the template code.

Now all the radio button CSS styles are around who the element looks, and all the positioning (margin, padding) are done with classes on the page the elements are on.

Reply:
Bullet points make sense :+1: 

Not sure I'm following this part:
> Now all the radio button CSS styles are around who the element looks

Reply:
_how_ the element _looks_, rather than how the elements as a whole are positioned onto the page. sorry typo!

Reply:
Ah, got it